man/workshop_activate.Rd

```diff
@@ -4,10 +4,13 @@
 \alias{workshop_activate}
 \title{Choose workshops and activate them}
 \usage{
-workshop_activate(df)
+workshop_activate(df, alert = "default")
```

Comment:
I find this a little bit awkward, I would make the default to be triggered when not supplying the 'alert' argument (which currently also works). But I guess it's just me coming from Python.

Reply:
Hmmm... Let me take a quick look. You are right, and I was about to explain why I made this choice, but I actually think there is a better way to do this. BRB 🙃 — don't merge yet!

Reply:
OK — I made a change now: the "default" (which was awkward indeed) is gone; the hardcoding of the name of the coordinator is now in the `workshop_activate()` function itself, which also makes it very easy to replace (that was the idea behind the "default"),  juuuuuuust in case the current training coordinator goes on, say, maternity leave.

Reply:
Yes, this looks nicer 👍